### PR TITLE
Sign up sign in form

### PR DESCRIPTION
**Pull request by ThomasDCW**
First working version of the frontend form Sign In & Sign Up, designed with material UI

### CODE REVIEW

**Review by ilalande — APPROVED** [review]

File: frontend/src/pages/LogIn/index.jsx [hunk 1]
```diff
@@ -1,13 +1,93 @@
...
+        <div className="tab">
+          <Box sx={{ width: "100%" }}>
+            <Box sx={{ borderBottom: 1, borderColor: "divider" }}>
+              <Tabs
+                centered
+                value={value}
+                onChange={handleChange}
+                aria-label="basic tabs example"
+              >
+                {/* eslint-disable-next-line react/jsx-props-no-spreading */}
```
Comment (LuigiMenez) [inline frontend/src/pages/LogIn/index.jsx:68]: Nop

File: frontend/src/pages/LogIn/index.jsx [hunk 2]
```diff
@@ -1,13 +1,93 @@
...
+                <Tab
+                  label="Se connecter"
+                  id={tabIndex(0).id}
+                  aria-controls={tabIndex(0)["aria-controls"]}
+                />
+                {/* eslint-disable-next-line react/jsx-props-no-spreading */}
```
Comment (LuigiMenez) [inline frontend/src/pages/LogIn/index.jsx:74]: Nop

File: frontend/src/pages/LogIn/index.jsx [hunk 3]
```diff
@@ -1,13 +1,93 @@
...
+    primary: {
+      main: "#1ea0a5",
+    },
+  },
+});
+
+function TabPanel(props) {
+  const { children, value, index } = props;
+
+  TabPanel.propTypes = {
```
Comment (LuigiMenez) [inline frontend/src/pages/LogIn/index.jsx:20]: The proptypes are placed at the end of the file

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Isn't it possible to refactor everything into several small components?
Is not disable EsLint

**Review by ilalande — DISMISSED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by Dash293 — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**